docs/ref/models/querysets.txt

```diff
@@ -1354,6 +1354,30 @@ one, doing so will result in an error.
     reader, is slightly faster and consumes a little less memory in the Python
     process.
 
+    For example::
+
+         # Both these models use the same underlying database table
+         # However, you would use an unmanaged model with fewer fields if they
+         # were the fields most often used.
+         class CommonlyUsedModal(models.Model):
+             f1 = models.CharField(max_length=10)
+             f2 = models.CharField(max_length=10)
+
+             class Meta:
+                 managed = False
+                 db_table = 'app_largetable'
+
+         class ManagedModel(models.Model):
+             f1 = models.CharField(max_length=10)
+             f2 = models.CharField(max_length=10)
+             f3 = models.CharField(max_length=10)
+
+             class Meta:
+                 db_table = 'app_largetable'
+
+         # Same as `ManagedModel.objects.all().defer("f3")` but more succinct
+         CommonlyUsedModal.objects.all()
+
```

Comment:
@charettes , it's not an example of how to accomplish this effect / functionality, it's specifically an example of how to avoid using defer() when possible.